grafana/scylla-manager.template.json

```diff
@@ -142,13 +142,6 @@
                               "dashversion":">2.4",
                               "refId": "A"
                             },
-                            {
-                              "expr": "sum(avg(scylla_manager_repair_progress{cluster=~\"[[cluster]]\", instance=\"\", keyspace=\"\"}) by (task) * sum(scylla_manager_task_active_count{cluster=~\"[[cluster]]\", type=\"repair\"}) by (task))  or vector(0)",
-                              "format": "time_series",
-                              "intervalFactor": 2,
-                              "dashversion":"<2.4",
-                              "refId": "A"
```

Comment:
You don't need to remove this part, if your change is for 3.0 this section is only valid for 2.3 or lower, that's what the dashversion is doing.

Reply:
I've returned back the expressions. I deleted them because they were existed in my dashboard which I built for version 3.0, but I did it directly execute python script, so, probably, somewhere I had mistake and the expressions appeared in the dashboard after building.